lms/djangoapps/shoppingcart/processors/helpers.py

```diff
@@ -21,10 +21,10 @@ def get_processor_config():
         settings.CC_PROCESSOR_NAME, {}
     )
 
-    # Check whether we're in a microsite that overrides our configuration
-    # If so, find the microsite-specific configuration in the 'microsites'
+    # Check whether configuration override exists,
+    # If so, find the configuration-specific cybersource config in the configurations.
     # sub-key of the normal processor configuration.
-    config_key = microsite.get_value('cybersource_config_key')
+    config_key = configuration_helpers.get_value('cybersource_config_key')
     if config_key:
         config = config['microsites'][config_key]
```

Comment:
@mattdrayer I am not much familiar with this code here, but it seems changing this would require configuration updates also